SailfishSDK.py

```diff
@@ -182,10 +182,9 @@ def maybe_install_sdk(self, engine_memory_size_mb=None):
             raise
 
         command = variables['${INSTALLER}']
-        args = variables['@{INSTALLER_ARGS}']
         build_engine_type = variables['${BUILD_ENGINE_TYPE}']
-        args += ['--verbose', 'non-interactive=1', 'accept-licenses=1',
-                'buildEngineType=' + build_engine_type]
+        args = variables['@{INSTALLER_ARGS}'] + ['--verbose', 'non-interactive=1',
+                'accept-licenses=1', 'buildEngineType=' + build_engine_type]
```

Comment:
I don't get it. The only difference I see here is that instead of initializing args first with variables['@{INSTALLER_ARGS}'] and then appending to it after figuring out the build engine type, we now assing the end result to it straight away. I don't see how the result could be any different?

Reply:
The original code does not make a copy of (the object bound to) `variables['@{INSTALLER_ARGS}']` - it just (re)binds it, so `args += ...` modifies the original object again and again every invocation.